R/plot_network.R

```diff
@@ -221,9 +221,7 @@ plot_network <- function(data_obj, marker_pairs = NULL, collapsed_net = TRUE,
     trait_locale <- which(trait %in% names(data_obj$max_var_to_pheno_influence))
     if(length(trait_locale) < length(trait)){
       not_found <- which(!trait %in% names(data_obj$max_var_to_pheno_influence))
-      cat("I couldn't find the following traits:")
-      cat(trait[not_found], sep = "\n")
-      return()
+      stop("I couldn't find the following traits:", paste(trait[not_found], collapse = ", "))
```

Comment:
in other places the message text is in a warning() and the stop is called.. shall it be changed here for consistency?

Reply:
Good call. I'd forgotten I did it the other way before.